### DISCUSSION TITLE
Need a template code to  add drop down menu ?

### DISCUSSION THREAD

**Discussion by ghost** [coqui-ai/TTS#833] [General Q&A] [answered]
This image is seen in documentation:
![image](https://user-images.githubusercontent.com/88183889/134320701-f18458d5-c807-4232-ab4d-c14059af4dbd.png)

but When I run in my local machine I don't have that drop down menu to choose speaker ?
As I am noob/beginner, I don't Know How to add the drop down- menu and list all available models... If anyone who knows how to make it.. kindly share me the template for it ... @erogol @thorstenMueller  and all community members ..

Thanks in Advance

**Comment by lexkoro** [comment, marked as answer]
You need to run the tts-server with a multi-speaker model for the drop-down to show

Try this:
`tts-server --model_name tts_models/en/vctk/vits`

**Reply by ghost** [reply]
Thanks @lexkoro
